src/js/pages/Setup.js

```diff
@@ -464,12 +476,49 @@ class Setup extends mixin(StoreMixin) {
           validationErrorText: this.getErrors('ip_detect_script'),
           value: this.state.formData.ip_detect_script
         }
-      ]
+      ],
+      {
+        fieldType: 'checkboxMultiple',
+        name: 'nested_cluster_options',
+        formRowType: 'inline',
+        value: [
+          {
+            name: 'telemetry_enabled',
+            label: this.getCheckboxLabel('Send Anonymous Telemetry', (<span>
```

Comment:
@mlunoe Just updated the tooltip content. Do you think we still need to have this `getCheckboxLabel` method? I don't think it's that beneficial to be honest.